src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml

```diff
@@ -1,76 +1,65 @@
 ﻿@model Esfa.Recruit.Employer.Web.ViewModels.Part1.Employer.EmployerViewModel
-
 <div class="grid-row">
     <div class="column-full">
 
         @Html.Partial("_Part1Header")
 
-        <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
-            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
-                Please fix these errors
-            </h2>
-        </div>
+        @Html.Partial("_ValidationSummary", new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames })
 
         <div id="section_location">
-            @if (Model.HasOnlyOneOrganisation)
-            {
-            <div>@Model.Organisations.First().Name</div>
-            }
+            
+            <div condition="@Model.HasOnlyOneOrganisation"><p>@Model.Organisations.First().Name</p></div>

```

Comment:
Should this be `asp-condition` so we know its required for server side processing?